lib/graphql/get_app_by_api_key.graphql

```diff
@@ -0,0 +1,11 @@
+query getApp($api_key: String!) {
+  app(apiKey: $api_key) {
+    id
+    title
+    apiKey
+    apiSecretKeys{
+      secret
+    }
+    appType
```

Comment:
Is this field used anywhere?

Reply:
I kept the same GraphQL response structure for `app` as was defined in `all_organizations`. My thought process was to keep it the same so other parts of the CLI could reuse the GraphQL with minimal effort. I didn't think an extra field or two on a single call would be enough to offset the reusability.

Reply:
I removed these extra fields since we moved the graphql to be an extension only call now.